### PR TITLE
Wait a finite amount of time to join the JvmMonitorThread

### PR DESCRIPTION
**Pull request by bradyoo**

### CODE REVIEW

**Comment by bradyoo** [comment]
@gpang can you please take a look? This is the main blocker for java 11 github-actions

**Comment by bradyoo** [comment]
@gpang

File: core/common/src/main/java/alluxio/util/JvmPauseMonitor.java [hunk 1]
```diff
@@ -93,7 +93,7 @@ public void stop() {
     }
     mJvmMonitorThread.interrupt();
     try {
-      mJvmMonitorThread.join();
+      mJvmMonitorThread.join(60 * 1000);
```
Comment (gpang) [inline core/common/src/main/java/alluxio/util/JvmPauseMonitor.java:96]: I don't think the jvm monitor is doing very much, so we probably don't need to wait so long. Maybe: `5 * Constants.SECOND_MS`?

Also, I think the thread while loop should be something like `while (!Thread.currentThread().isInterrupted())`

**Review by gpang — CHANGES_REQUESTED** [review]
@bradyoo I left a comment

**Comment by bradyoo** [comment]
@gpang's suggested fix reflected. Though !Thread.currentThread().isInterrupted() didn't end up working.

**Review by gpang — APPROVED** [review]
LGTM

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Comment by LuQQiu** [comment]
alluxio-bot, merge this to branch-2.4 please

**Comment by LuQQiu** [comment]
alluxio-bot, cherry-pick this to branch-2.4 please

**Pull request merged**